### PR TITLE
Added cleanup script for service accounts

### PR DESCRIPTION
**Pull request by radtriste**
Troubleshoot script in case limit exceeded for service accounts

Please make sure that your PR meets the following requirements:

- [ ] Your code is properly formatted according to [this configuration](https://github.com/kiegroup/kogito-runtimes/tree/main/kogito-build/kogito-ide-config)
- [ ] Your commit messages are clear and reference the JIRA issue e.g: "[MGDOBR-1] - $clear_explanation_of_what_you_did"
- [ ] All new functionality is tested
- [ ] Pull Request title is properly formatted: `MGDOBR-XYZ Subject`
- [ ] Pull Request contains link to the JIRA issue
- [ ] Pull Request contains link to any dependent or related Pull Request
- [ ] Pull Request contains description of the issue
- [ ] Pull Request does not include fixes for issues other than the main ticket

<details>
<summary>
How to trigger pipelines and use the bots:
</summary>

* <b>Run the end to end pipeline</b>  
  Annotate the pull request with the label: `safe to test`. If you want to run the pipeline again, remove and add it again.

* <b>Rebase the pull request</b>  
  Comment with: `/rebase`.

</details>

### CODE REVIEW

**Review by jiripetrlik — COMMENTED** [review]
Would it be possible to read username from rhoas whoami or from localconfig?

**Comment by radtriste** [comment]
> Would it be possible to read username from rhoas whoami or from localconfig?

Good idea. Will set that in place with priorities:

cmdline arg > localconfig > rhoas whoami > $USERNAME

File: dev/bin/troubleshoot/cleanup-service-accounts.sh [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+echo "Using owner matching ${username}"
+
+ids=$(ocm get /api/kafkas_mgmt/v1/service_accounts --parameter size=1000 | jq '.items[] | {id,owner} | join(" ")' | grep ${username} | tr -d '"' | awk '{print $1}')
+
+# for id in $ids
+# do
+#     ocm delete /api/kafkas_mgmt/v1/service_accounts/${id}
+#     echo "Deleted service account with id ${id}"
+# done
```
Comment (jomarko) [inline dev/bin/troubleshoot/cleanup-service-accounts.sh:34]: please do not forget uncomment

**Review by jomarko — COMMENTED** [review]
thank you for this PR @radtriste

File: dev/bin/troubleshoot/cleanup-service-accounts.sh (hunk 1 above)
Comment (radtriste) [reply dev/bin/troubleshoot/cleanup-service-accounts.sh]: done :)

**Review by jomarko — APPROVED** [review]
Thank you @radtriste

**Pull request merged**